packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md

```diff
@@ -539,7 +539,7 @@ Content-Type: text/json;charset=UTF-8
 
 **PATCH** /Groups/*$groupID*
 
-Updates to existing group names for existing Okta groups are handled by a PATCH operation. The group must already be pushed out to the SCIM server.
+Updates to existing group names for existing Okta groups are handled by a PATCH operation. The group must already be pushed out to the SCIM server and Group Push Enhancements feature must be enabled in the Okta org together together with `Rename groups` option under Push Groups tab.
```

Comment:
Updates to existing group names for existing Okta groups are handled by a PATCH operation. The group must be already pushed out to the SCIM server. Also, the Group Push Enhancements feature must be enabled in your Okta org, and the **Rename groups** option must be selected in the **Group Push Settings** on the **Push Groups** tab.